File: programmers/기존풀이/소수만들기.js
function findPrimeNum(num) {
    for (let i = 2 ; i <= Math.sqrt(num); i++) {
        if (num % i === 0)
            return false
    }
    return true
}

function solution(nums) {
    let answer = 0;
    for(let i = 0 ; i < nums.length-2; i++) {
        for(let j = i + 1; j < nums.length-1; j++) {
            for(let k = j + 1; k < nums.length; k ++) {
                if(findPrimeNum(nums[i] + nums[j] + nums[k])) {
                    answer ++
                }ㅏ
            }
        }
    } 
    return answer;
}

File: sparta/소수만들기.py
from itertools import combinations, permutations

def isPrime(num):
    cnt = 1
    for i in range(1, num // 2 + 1):
        if num % i == 0:
            cnt += 1
    return cnt

def solution(nums):
    result = 0
    perm = list(combinations(nums, 3))
    for i in perm:
        if isPrime(sum(i)) < 3:
            result += 1
    return result